site_scons/site_tools/compiler_setup.py

```diff
@@ -121,6 +121,11 @@ def _base_setup(env):
 
     env['BSETUP'] = compiler
 
+    if GetOption('code_coverage'):
+        env.AppendUnique(CXXFLAGS=['-fprofile-arcs', '-ftest-coverage'])
+        env.AppendUnique(CCFLAGS=['-fprofile-arcs', '-ftest-coverage'])
+        env.AppendUnique(CGO_CFLAGS=['-fprofile-arcs', '-ftest-coverage'])
```

Comment:
I don't think this has any effect.  It needs to be AppendENVPath('CGO_CFLAGS', ...) similar to CGO_LDFLAGS below

Reply:
Jeff, Thanks for the review and comments.
        I comment-out above lines and started a scons-3 build, however observe following error:
scons: *** [build/release/gcc/src/control/daos] CalledProcessError : Command '['/usr/bin/go', 'list', '--json', '-mod=vendor', 'cmd/daos/acl.go']' returned non-zero exit status 1.
scons: building terminated because of errors.